pkg/config/consts.go

```diff
@@ -35,6 +35,12 @@ const (
 
 	// KadcastInitialHeight sets the default initial height for Kadcast broadcast algorithm.
 	KadcastInitialHeight byte = 128
+
+	// The dusk-blockchain executable version.
+	NodeVersion = "0.5.0"
```

Comment:
In the future, this should be called `Chain` instead of `Node` to avoid any confusion

Reply:
This can be the object of a minor followup